discord_music_bot/config.py

```diff
@@ -20,4 +20,8 @@
         "client_secret": env.str("CLIENT_SECRET"),
     }
 
+with env.prefixed("OUTRO_"):
```

Comment:
you can change `OUTRO_` TO `OUTRO_VIDEO_` since both variables include this prefix